README.md

```diff
@@ -29,3 +29,7 @@ Note: "Rustaceans" is a term often used to refer to enthusiasts and users of the
 - [🏋️‍♂️ Rust programming excercises that help you learn new concepts](https://exercism.org/tracks/rust)
 - [👩‍🍳 Rust Cookbook](https://exercism.org/tracks/rust)
 - [💪 Rust Gym: Leetcode solutions, Advent of Code Solutions in Rust and more](https://rustgym.com/)
+- [🤖 The Crust of Rust](https://www.youtube.com/playlist?list=PLqbS7AVVErFiWDOAVrPt7aYmnuuOLYvOa)
+- [🦿 A half-hour to learn Rust](https://fasterthanli.me/articles/a-half-hour-to-learn-rust)
```

Comment:
@agnxsh also for this one, please create a Important Article section and then add this link.

something like
## Important Articles
Add the links here....

Reply:
check now